assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -160,6 +161,35 @@ export function registerDefaults( widgetsAPI ) {
 			},
 			CONTEXT_MAIN_DASHBOARD_TRAFFIC
 		);
+
+		widgetsAPI.registerWidget(
+			'audienceConnectAnalyticsCTA',
+			{
+				Component: ConnectAnalyticsCTATileWidget,
+				width: widgetsAPI.WIDGET_WIDTHS.FULL,
+				priority: 1,
+				wrapWidget: false,
+				modules: [ 'analytics-4' ],
+				isActive: ( select ) => {
+					const isAnalyticsConnected =
+						select( CORE_MODULES ).isModuleConnected(
+							'analytics-4'
+						);
+
+					/**
+					 * TODO: This widget should be shown only if the audience group
+					 * is set up for the current user. This should be fixed once
+					 * the audience settings become accessible without `analytics-4`
+					 * module being connected.
+					 * See: https://github.com/google/site-kit-wp/issues/8810 for
+					 * more details.
+					 */
+
+					return ! isAnalyticsConnected;
+				},
+			},
+			[ AREA_MAIN_DASHBOARD_TRAFFIC_AUDIENCE_SEGMENTATION ]
+		);
```

Comment:
Shouldn't module-specific widgets be placed in the module's `registerWidgets` function, in this case, located in `assets/js/modules/analytics-4/index.js`? Or are we placing it here to cover the case where Analytics isn't available as a module at all (possible via the `googlesitekit_available_modules` filter)?

Reply:
Well I think I've basically followed the instruction from IB without thinking about it too much. While putting this here for when the module assets are not loaded makes sense, the actual CTA will not really function without the assets, so I'll move this to the module specific location instead. Thanks